include/nihstro/shader_bytecode.h

```diff
@@ -225,7 +225,7 @@ struct DestRegister {
 };
 
 struct OpCode {
-    enum class Id : uint32_t {
+    enum Id : uint32_t {
```

Comment:
Bleh, with this change there's no need to keep around the Id name of that enum at all (I'd rather not have inconsistent usage of `Id::ADD` and `ADD` mixed just because it's possible).

Unless there is some critical reason for doing this, either fix the JIT to use `Id::` prefixes or submit further changes to remove the `Id::` prefixes from nihstro (and consequently make this enum unnamed).